contrib/nextjs/next.bazel.mjs

```diff
@@ -12,6 +12,10 @@ if (nextjsStandaloneConfig.startsWith(process.env.BAZEL_BINDIR)) {
         process.env.BAZEL_BINDIR.length + 1
     )
 }
+
+// Support invocation from another workspace: strip "external/<repo>/" if present
+nextjsStandaloneConfig = nextjsStandaloneConfig.replace(/^external\/[^/]+\//, '')
```

Comment:
This is needed despite your fix? Or does your main fix just allow this one to be done?